src/actions/triggerBindings.test.js

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as API from '../api';
+import * as creators from './actionCreators';
+import { fetchTriggerBinding, fetchTriggerBindings } from './triggerBindings';
+
+it('fetchTriggerBinding', async () => {
+  jest.spyOn(creators, 'fetchNamespacedResource');
+  const name = 'TriggerBindingName';
+  const namespace = 'namespace';
+  fetchTriggerBinding({ name, namespace });
+  expect(creators.fetchNamespacedResource).toHaveBeenCalledWith(
+    'TriggerBinding',
+    API.getTriggerBinding,
+    { name, namespace }
+  );
+});
+
+it('fetchTriggerBindings', async () => {
+  jest.spyOn(creators, 'fetchNamespacedCollection');
+  const namespace = 'namespace';
+  fetchTriggerBindings({ namespace });
+  expect(creators.fetchNamespacedCollection).toHaveBeenCalledWith(
+    'TriggerBinding',
+    API.getTriggerBindings,
+    { namespace }
```

Comment:
nit: Please include `filters` in the test (this will be consistent with [`pipelineRuns.test`](https://github.com/tektoncd/dashboard/blob/master/src/actions/pipelineRuns.test.js#L33-L38).

Reply:
Done